install/outline-install.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 community-scripts ORG
+# Author: Gerhard Burger (burgerga)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+read -r -p "Enter the public url for your Outline instance (e.g., https://outline.your-domain.tld)): " URL
+read -r -p "Would you like to add Adminer? <y/N> " adminer_prompt
```

Comment:
Yes, I made a note about this in the PR commend at the top. Happy to change this back if required. I just think this would be a better user experience.

Reply:
Follow conventions for now, we prompt before install step for instant feedback on your prompt. Maybe we can improve in the future